packages/terra-form-input/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+ * Removed className `form-input` when `type="file"`.
```

Comment:
It would be generally easier to understand what is changed if description is something like what you have on the PR title:

`Update input to be native input when the type is file`

Reply:
Updated here https://github.com/cerner/terra-core/pull/3386/commits/fda676b3d846449ad4561fa467c7cbbaabb8bd97